src/main/webapp/library.js

```diff
@@ -0,0 +1,22 @@
+import { Sources } from './sources'
+import Workspaces, { Workspace } from './workspaces/workspaces'
+import SimpleSearch from './simple-search'
+import Search from './search/search'
+import SearchResults from './search/results'
+import SearchDetails from './search/details'
+import ResultForms from './result-forms'
+import SearchForms from './search-forms'
+import { About } from './about'
+
+export {
+  About,
+  Sources,
+  Workspace,
+  Workspaces,
+  SimpleSearch,
+  Search,
+  SearchResults,
+  SearchDetails,
+  ResultForms,
+  SearchForms,
```

Comment:
Investigate why SearchForms is causing issues - probably something to do it being typescript